src/ResourceManager/Automation/Commands.Automation/ChangeLog.md

```diff
@@ -18,12 +18,6 @@
         - Additional information about change #1
 -->
 ## Current Release
-
-## Version 5.1.1
-* Fixed issue with default resource groups not being set.
-* Updated common runtime assemblies
-
-## Version 5.1.0
```

Comment:
I've added previous entries back removed by the merged. I have also included an entry for this release. I am bumping  from version 5.1.1 to 6.0.0 since this is a major release (switching from Hyak to Swagger based SDK). 

# Version 6.0.0
* Swagger based Azure Automation cmdlets
* Added Update Management cmdlets
* Added Source Control cmdlets